Library/Formula/pmd.rb

```diff
@@ -7,12 +7,13 @@ class Pmd < Formula
   depends_on "maven" => :build
 
   def install
+    ENV["_JAVA_OPTIONS"] = "-Duser.home=#{ENV["HOME"]}”
```

Comment:
This has been closed with the wrong form of `"`. You've used `”` rather than `"`, which upsets Ruby :smile: